trade_remedies_caseworker/core/utils.py

```diff
@@ -196,16 +196,17 @@ def notify_footer(api_client, email=None):
     return default_footer
 
 
-def notify_contact_email(api_client, case_number=None):
+def notify_contact_email(api_client, case_number=None, notify_sys_param_name=None):
```

Comment:
can you please add type hinting, e.g.

```suggestion
def notify_contact_email(api_client: Client(), case_number: uuid=None, notify_sys_param_name: str=None) -> bool:
```